src/Files.App.Win32/NativeMethods.json

```diff
@@ -0,0 +1,4 @@
+{
+	"$schema": "https://aka.ms/CsWin32.schema.json",
+	"public": true
+}
```

Comment:
Can you add `"allowMarshaling": false` as well? We want to make sure we don't use any runtime marshalling, especially in the server.

Reply:
I added it in the file in Server project.
Do we want to override the setting for every project?

Reply:
I don't think you need to add this configuration in other projects. The generated types will be public in Files.App.Win32, and can be consumed directly by other projects.

Reply:
Sure

Reply:
public: true
doesn't work...